internal/bminventory/inventory_test.go

```diff
@@ -2148,8 +2150,9 @@ var _ = Describe("KubeConfig download", func() {
 		clusterID = strfmt.UUID(uuid.New().String())
 		mockS3Client = s3wrapper.NewMockAPI(ctrl)
 		mockJob = job.NewMockAPI(ctrl)
+		dummy := &leader.DummyElector{}
```

Comment:
why do you need it if it's not used or tested?

Reply:
it is the same as mock, i can't set none cause monitor expects it but there is no need to check it in every test

Reply:
so it is mock that is doing noting

Reply:
how inventory calls for monitor?